app/javascript/packs/articlePage.jsx

```diff
@@ -47,7 +48,18 @@ if (shareDropdownButton.dataset.initialized !== 'true') {
     document
       .querySelectorAll('#article-show-more-dropdown [href]')
       .forEach((link) => link.addEventListener('click', closeDropdown));
+
+    // Add Ahoy stats for dropdown share options
+    const container = document.getElementsByClassName(
+      'crayons-article-actions__inner',
+    )[0];
```

Comment:
suggestion (not blocking) - you could use `querySelector` which will stop looking after it finds the first one. In practice it doesn't matter much though, as there will only be one on the page anyway 😄 

```suggestion
    const container = document.querySelector(
      '.crayons-article-actions__inner',
    );
```